R/sim_core_analytic.R

```diff
@@ -2,16 +2,17 @@
 #' Mostly useful for estimations / optimal design. Has no checks (for speed)!
 #' This is separate function for analytic equations only. See `sim_core()` for ODE function.`
 #'
+#' @inheritParams sim
 #' @param sim_object list with design and simulation parameters
 #' @param model model object
 #' @param duplicate_t_obs allow duplicate t_obs in output? E.g. for optimal design calculations when t_obs = c(0,1,2,2,3). Default is FALSE.
 #' @param t_init time of initization of the ODE system. Usually 0.
 #' @export
-sim_core_analytic <- function(
-  sim_object = NULL,
-  model,
-  duplicate_t_obs = FALSE,
-  t_init = 0) {
+sim_core_analytic <- function(ode,
```

Comment:
Added `ode` as an argument here, otherwise r cmd check complains that it's not found. `sim()` takes `ode` as an arg so I guessed it should be the same here, but not certain.

Reply:
hmm, that is a bit surprising. I'm ok with committing this for now, will look at this when I get to RXR-633 later this week / next.